scripts/themes/index.js

```diff
@@ -21,16 +21,21 @@ const { value: themeMap } = renderSync({
 const themes = Object.keys(themeMap);
 
 // Generates the data for themes and tokens to template with Handlebars.
-const tokens = Object.keys(themeMap[themes[0]].value).map(token => ({
-  themes: [],
-  token,
-}));
+const tokens = Object.keys(themeMap[themes[0]].value)
+  // TODO - currently only handling color tokens.
+  .filter(token => themeMap[themes[0]].value[token].type === 'SassColor')
```

Comment:
I'd like to pull this out into a helper method above for reuse here and on line 38 - For example:

`const filterColor = ({ type }) => type === 'SassColor';`

And its usage - `.filter(token => filterColor(themeMap[themes[0]].value[token]))`

Reply:
I'm having trouble seeing the value of abstraction here. `type === 'SassColor'` in the location it is used (to avoid looking elsewhere, to see what is actually being filtered for) is not preferable to having a method called `filterColor`?

Reply:
I think alternatively to reuse, it might be more appropriate to find an alternative solution where the colors are filtered before they get to these points to avoid duplication

Reply:
Should I add a ‘TODO’ as well for additional work to be done? 😅